pages/02_picture.py

```diff
@@ -31,22 +45,39 @@ def get_prompt(filepath):
         return f.read()
 
 
```

Comment:
- こんな感じの、写真を見せるコンテナをつくってもらえると、都度表示しなくていけます。
```suggestion
def show_png(image_data):
    """ 写真表示 """
    st.image(image_data)


```